plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023 Ansible Project
+# Copyright (c) 2023 Gomathi Selvi Srinivasan (@GomathiselviS)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+module: rds_global_cluster_info
+version_added: 6.6.0
+short_description: Obtain information about Aurora global database clusters
+description:
+  - Obtain information about Aurora global database clusters.
+options:
+    global_cluster_identifier:
+        description:
+          - The user-supplied Global DB cluster identifier.
+          - If this parameter is specified, information from only the specific DB cluster is returned.
+          - This parameter is not case-sensitive.
+          - If supplied, must match an existing DBClusterIdentifier.
+        type: str
+    filters:
+        description:
+            - A filter that specifies one or more DB clusters to describe.
+              See U(https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_Filter.html).
+        type: dict
+author:
+  - Gomathi Selvi Srinivasan (@GomathiselviS)
+extends_documentation_fragment:
+  - amazon.aws.common.modules
+  - amazon.aws.region.modules
+  - amazon.aws.boto3
+"""
+
+EXAMPLES = r"""
+- name: Get info of all existing DB clusters
+  amazon.aws.rds_global_cluster_info:
+  register: _result_cluster_info
+
+- name: Get info on a specific DB cluster
+  amazon.aws.rds_global_cluster_info:
+    global_cluster_identifier: "{{ cluster_id }}"
+  register: _result_global_cluster_info
+
+- name: Get info all DB clusters with specific engine
+  amazon.aws.rds_cluster_info:
+    filters:
+      engine: "aurora-mysql"
+  register: _result_global_cluster_info
+"""
+
+RETURN = r"""
+global_clusters:
```

Comment:
if you keep the input identifier as a single string, you should set the output as a list of clusters info

Reply:
I donot understand this comment. The argument `global_cluster_identifier` is defined as a string, while the return value is stored in a list named `global_clusters`. This aligns with the documentation provided by AWS Boto3 and is consistent with the existing RDS info module we have in place.